crates/ibc/src/core/ics04_channel/commitment.rs

```diff
@@ -69,3 +69,52 @@ impl From<Vec<u8>> for AcknowledgementCommitment {
         Self(bytes)
     }
 }
+
+pub use compute::{compute_ack_commitment, compute_packet_commitment};
+
+pub mod compute {
+    use super::*;
+    use crate::core::ics04_channel::msgs::acknowledgement::Acknowledgement;
+    use crate::core::ics04_channel::timeout::TimeoutHeight;
+    use crate::timestamp::Timestamp;
+
+    /// Compute the commitment for a packet.
+    ///
+    /// Note that the absence of `timeout_height` is treated as
+    /// `{revision_number: 0, revision_height: 0}` to be consistent with ibc-go,
+    /// where this value is used to mean "no timeout height":
+    /// <https://github.com/cosmos/ibc-go/blob/04791984b3d6c83f704c4f058e6ca0038d155d91/modules/core/04-channel/keeper/packet.go#L206>
+    pub fn compute_packet_commitment(
```

Comment:
I think we can make this `pub(crate)` since it is only meant to be used internally by ibc-core.

Reply:
[587fa8b](https://github.com/cosmos/ibc-rs/pull/492/commits/587fa8bab4b78dd48f77a3af70f1954943164e44)